### PR TITLE
Concurse-fly: Add version 5.5.1

### PR DESCRIPTION
**Pull request by javakhishvili**
Concourse is primarily driven from the command-line; there is no GUI config wizard. So, the first step to getting started with Concourse is to install the fly CLI tool.

### CODE REVIEW

File: bucket/fly.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "version": "5.5.1",
+    "description": "Command-line tool used to control Concourse CI/CD",
+    "license": "Apache-2.0",
+    "homepage": "https://concourse-ci.org/",
```
Comment (Ash258) [inline bucket/fly.json]:
```suggestion
    "homepage": "https://concourse-ci.org/fly.html",
```

File: bucket/fly.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+    "version": "5.5.1",
+    "description": "Command-line tool used to control Concourse CI/CD",
```
Comment (Ash258) [inline bucket/fly.json]:
```suggestion
    "description": "Concourse CI/CD manager and controller.",
```

**Comment by linsui** [comment]
Should we rename it as concurse-fly? Fly is a common word.

**Comment by javakhishvili** [comment]
> Should we rename it as concurse-fly? Fly is a common word.

Yes, that makes perfect sense. Agreed.

**Pull request merged**